crates/node/src/main.rs

```diff
@@ -23,6 +23,9 @@ fn main() {
             let handle = builder
                 .node(ScrollRollupNode::new(args))
                 .launch_with_fn(|builder| {
+                    // Log ScrollChainConfig during node launch as requested in the issue
+                    let chain = &builder.config().chain;
+                    info!(target: "reth::cli", "ScrollChainConfig: {:?}", chain);
```

Comment:
No need for the comment, can you make the suggested changes and move this after line 44 so it gets logged after launch?
```suggestion
                    let config = &builder.config().chain.config;
                    info!(target: "reth::cli", ?config, "Running with config");
```